contents/docs/data/anonymous-vs-identified-events.mdx

```diff
@@ -107,6 +107,8 @@ import HowToCaptureIdentifiedEventsFlutter from '../product-analytics/_snippets/
 
 ## Frequently asked questions
 
+<!---  General questions -->
```

Comment:
These headings are useful. Let's change all of these to actual headings our users will see.

Note we'll need to modify all question headings to add an extra "#"
```suggestion
### General questions
```